Sub-menu-via-event-target/test.js

```diff
@@ -0,0 +1,20 @@
+const ul = document.querySelector(".tree");
+
+const li = ul.querySelectorAll("li");
+
+li.forEach((item) => {
+  const span = document.createElement("SPAN");
+  span.classList.add("span");
```

Comment:
Please call the class more readable and understandable.
Now the tag is span, the variable is span, and the class is called span. This can be confusing.